web-client/src/pages/signup.js

```diff
@@ -0,0 +1,65 @@
+import '../components/signup.css';
+import '../global.css';
+
+export default function signup(){
```

Comment:
Component names should always use PascalCase

```suggestion
export default function Signup(){
```

Reply:
Ahh yes i forgot !